providers/implementations/storemgmt/file_store.c

```diff
@@ -771,6 +772,25 @@ static int file_close(void *loaderctx)
     return 1;
 }
 
+static void *file_open_ex(void *provctx, const char *uri,
+                          const OSSL_PARAM params[],
+                          OSSL_PASSPHRASE_CALLBACK *pw_cb, void *pw_cbarg)
+{
+    struct file_ctx_st *ctx = NULL;
+
+    ctx = file_open(provctx, uri);
+    if (ctx == NULL)
+        return NULL;
+
+    if (!file_set_ctx_params(ctx, params)) {
+        ERR_raise(ERR_LIB_PROV, ERR_R_PROV_LIB);
+        free_file_ctx(ctx);
+        return NULL;
+    }
+
+    return ctx;
+}
+
```

Comment:
Not sure I understand why implementing this change. And I also thought that when we have the `open_ex` variant, it's worth explicitly implement `open` via `open_ex`, not this way...

Reply:
It was mostly to test things, but it is unneeded in the file store unless it will grow a way to decrypt encrypted pem files (or maybe we grow support to use p12 files directly?).

Anyway I will drop this change on rebase